keps/sig-network/1880-multiple-service-cidrs/README.md

```diff
@@ -882,81 +818,60 @@ See Drawbacks section
 
 When creating a Service this will require to create an IPAddress object,
 previously we updated a bitmap on etcd, so we keep the number of request
-but the size is reduced considerable.
+but the size of the objects stored is reduced considerable.
 
 ###### Will enabling / using this feature result in introducing new API types?
 
 See [API](#api)
 
 ###### Will enabling / using this feature result in any new calls to the cloud provider?
 
-N/A
+No
 
 ###### Will enabling / using this feature result in increasing size or count of the existing API objects?
 
-See [Drawbacks](#drawbacks)
+See [Drawbacks](#drawbacks), a new IPAddress object will be created by Service.Spec.ClusterIP
 
 ###### Will enabling / using this feature result in increasing time taken by any operations covered by existing SLIs/SLOs?
 
-<!--
-Look at the [existing SLIs/SLOs].
-
-Think about adding additional work or introducing new steps in between
-(e.g. need to do X to start a container), etc. Please describe the details.
-
-[existing SLIs/SLOs]: https://git.k8s.io/community/sig-scalability/slos/slos.md#kubernetes-slisslos
--->
+No
 
 ###### Will enabling / using this feature result in non-negligible increase of resource usage (CPU, RAM, disk, IO, ...) in any components?
 
-The apiservers will increase the memory usage because they have to keep a local informer with the new objects ServiceCIDR and IPAddress.
+The apiservers and cloud-controller manager will increase the memory usage because they have to keep a local informer with the new objects ServiceCIDR and IPAddress.
```

Comment:
```suggestion
The apiservers and kube-controller manager will increase the memory usage because they have to keep a local informer with the new objects ServiceCIDR and IPAddress.
```